scania-digital-design-system/tegel
# Feat/header filter

**Describe pull-request**  
Adds functionality in header for filtering columns

**How to test**  
1. Go to  storybook
2. Check in Table > Header Filter

**Suggested test steps**

**Screenshots**  
![image](https://github.com/scania-digital-design-system/tegel/assets/8792533/f30ad873-247b-44e0-a67e-516792ac995d)
